### DISCUSSION TITLE
Memory Leak

### DISCUSSION THREAD

**Discussion by En-JON-eer** [Live-Charts/LiveCharts2#1169] [Q&A]
![image](https://github.com/beto-rodriguez/LiveCharts2/assets/89985196/f98d2cd6-9fd7-462d-b478-fd74e637014d)

I'm not sure why this is happening, but my application memory compounds over time. I am automatically updating the chart every time a new datapoint streams in. At low sample rate and high sample rate, the issue persists.

The main offenders in my memory stack are LiveChartsCore properties. Is there a way to mitigate this compounding memory? I confine the maximum allowable points on my chart to anywhere between 100 and 3000 points at any given time. 

I remove tail end points using observablePoints.RemoveAt(0); until the chart reaches the maximum displayed point count configured by my application settings. So it's not like I'm just plotting points infinitely causing run-away memory allocation.

Thanks in advance!
Jon

**Comment by beto-rodriguez** [comment]
which is your LiveCharts version?

**Reply by En-JON-eer** [reply]
![image](https://github.com/beto-rodriguez/LiveCharts2/assets/89985196/e313c34e-af4b-4766-8de3-e8f1c0b74d9b)

**Reply by beto-rodriguez** [reply]
that is a known issue and it is fixed, please update to beta.911 and the issue should be gone.

**Reply by En-JON-eer** [reply]
Thank you for the prompt response! I will look out for that update.

**Reply by En-JON-eer** [reply]
Bump. I am still seeing the issue on BETA.950

**Reply by beto-rodriguez** [reply]
Could it be an issue on your side? everything seems stable on the samples provided in the library:

![memoryleak](https://github.com/beto-rodriguez/LiveCharts2/assets/10853349/cf77b3ec-09da-462a-beca-f20e0ef146f3)

But feel free to create a repository/example where we can replicate the issue.
